src/Connection/RelayConnection.php

```diff
@@ -124,14 +124,6 @@ public function getClient()
         return $this->client;
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    protected function getIdentifier()
-    {
-        return $this->client->endpointId();
```

Comment:
In the context of connection this method doesn't need at all, cause it's just used to get a unique identifier for connection used on the client side. Here in RelayConnection this method requires connection to server, before actual connection is established.